.env

```
@@ -1,2 +1,2 @@
-GEMINI_API_KEY=YOUR_GEMINI_API_KEY
-DEEPGRAM_API_KEY=YOUR_DEEPGRAM_API_KEY
+REACT_APP_GEMINI_API_KEY=YOUR_GEMINI_API_KEY
+REACT_APP_DEEPGRAM_API_KEY=YOUR_DEEPGRAM_API_KEY
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REACT_APP_DEEPGRAM_API_KEY key should go before the REACT_APP_GEMINI_API_KEY key

(UnorderedKey)